### PR TITLE
Add other.test_function. NFC

### PR DESCRIPTION
**Pull request by sbc100**
This "test" file is only currently referenced by the docs.  For now
add a test to ensure it at least compiles.

### CODE REVIEW

File: tests/test_other.py [hunk 1]
```diff
@@ -11179,3 +11179,11 @@ def test_wasmfs_readfile(self):
   def test_hello_world_above_2gb(self):
     self.run_process([EMCC, test_file('hello_world.c'), '-sGLOBAL_BASE=2147483648', '-sINITIAL_MEMORY=3GB'])
     self.assertContained('hello, world!', self.run_js('a.out.js'))
+
+  def test_hello_function(self):
```
Comment (kripken) [inline tests/test_other.py:11183]:
```suggestion
  def test_docs_hello_function(self):
```

Comment (kripken) [reply tests/test_other.py:11183]: Maybe a convention of calling tests that keep docs valid `test_doc_` makes sense? Just an idea.

**Review by kripken — APPROVED** [review]
Maybe

File: tests/test_other.py (hunk 1 above)
Comment (sbc100) [reply tests/test_other.py:11183]: I'm not sure... maybe if we had more than one example of this.   My inclination is to instead move them into and `examples/` directory outside of tests completely.

**Pull request merged**